virtues/static/virtues/base.css

```diff
@@ -1,58 +1,188 @@
-@import url('https://fonts.googleapis.com/css2?family=Langar&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Merienda&family=Redressed&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Abel&family=Anton&family=Livvic:wght@300&display=swap');
 
-:root {
-    --main-color: rgb(44, 0, 114);
-    --second-color:  rgb(126, 68, 219);
-    --border-color: rgb(180, 180, 180);
-    --mainbg-color: black;
-    --mainfont-color: honeydew;
-    --headerfooter-font: 'Langar', sans-serif;
-    --body-font: 'Merienda', cursive;
+
```

Comment:
Apply a single space between selector and { following the project's style. E.g -
a {
/* code */
}